offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -105,6 +107,7 @@ class OfferToPDFConverter implements OfferExporter {
             return this.name;
         }
     }
+    private final Map productGroupClasses = [:]
```

Comment:
I think you could even initialize the productGroup map here. Its independend of the method call

```suggestion
    private final Map productGroupClasses = setProductGroupMapping()
```